averages.js

```diff
@@ -0,0 +1,70 @@
+// module.exports = {
```

Comment:
I wasn't sure how to export an Array.prototype extension in order to test it. Can it be done using the module.exports = {}  syntax?